.zuul.yaml

```diff
@@ -1,6 +1,36 @@
 ---
+- job:
+    name: functional-tests-on-osp18
+    parent: telemetry-operator-multinode-autoscaling
+    description: |
+      Run the stf functional tests on osp18
+    roles:  # adds in dependent roles i.e. put it in the role path
+      - zuul: github.com/openstack-k8s-operators/ci-framework
+    required-projects:
+      - name: github.com/infrawatch/service-telemetry-operator
+      - name: openstack-k8s-operators/ci-framework
+        override-checkout: main
+      - name: github.com/openstack-k8s-operators/dataplane-operator
+        override-checkout: main
+      - name: github.com/openstack-k8s-operators/infra-operator
+        override-checkout: main
+      - name: github.com/openstack-k8s-operators/openstack-operator
+        override-checkout: main
+      - name: github.com/openstack-k8s-operators/openstack-must-gather
+        override-checkout: main
+      - name: github.com/openstack-k8s-operators/telemetry-operator
```

Comment:
This is the replacement for STF - are we intending to use this code to actually test Telemetry Operator?

Reply:
We're going to add more tests as well, and use these feature-verification tests for both telemetry-operator and tripleo deployments.